apps/delivery-options/src/composables/useLanguage.ts

```diff
@@ -16,7 +16,8 @@ interface UseLanguage {
 
   setLocale(locale: string): void;
   setStrings(strings: Record<string, string>): void;
-  translate(translatable: AnyTranslatable): string;
+
+  translate(translatable: string): string;
```

Comment:
Why did you change this? Now there are type errors in places where this is called with a `Translatable`